spec/fixtures/site_settings/locale_override.yml

```diff
@@ -0,0 +1,14 @@
+basic:
+  default_locale:
+    default: 'en'
+    enum: 'LocaleSiteSetting'
+
+search:
+  min_search_term_length:
+    client: true
+    default:
+      development: 1
```

Comment:
Then I'll just push the existing overrides under `config/initializers/005-site_settings.rb`. And what about `hidden` property? There are only 4 settings under `developer` which are different than default. I guess it's not a problem to see those in development or test either. I can also do a sweep by the way.